### PR TITLE
magento/devdocs#: Manage the 'My Account' dashboard navigation links (Magento v.2.3)

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) extends https://github.com/magento/devdocs/pull/4672 in part Magento 2.3 because https://github.com/magento/devdocs/pull/4672 affects Magento 2.2 version only.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-manage.html

## Links to Magento source code

**Community Edition**
- [customer-account-navigation-account-link](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Customer/view/frontend/layout/customer_account.xml#L23)
- [customer-account-navigation-address-link](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Customer/view/frontend/layout/customer_account.xml#L35)
- [customer-account-navigation-account-edit-link](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Customer/view/frontend/layout/customer_account.xml#L42)
- [customer-account-navigation-downloadable-products-link](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Downloadable/view/frontend/layout/customer_account.xml#L11)
- [customer-account-navigation-newsletter-subscriptions-link](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Newsletter/view/frontend/layout/customer_account.xml#L11)
- [customer-account-navigation-billing-agreements-link](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Paypal/view/frontend/layout/customer_account.xml#L11)
- [customer-account-navigation-product-reviews-link](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Review/view/frontend/layout/customer_account.xml#L11)
- [customer-account-navigation-orders-link](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Sales/view/frontend/layout/customer_account.xml#L11)
- [customer-account-navigation-my-credit-cards-link](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Vault/view/frontend/layout/customer_account.xml#L11)
- [customer-account-navigation-wish-list-link](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Wishlist/view/frontend/layout/customer_account.xml#L11)

**Enterprise Edition**
- [customer-account-navigation-checkout-sku-link](https://github.com/magento/partners-magento2ee/blob/2.3.2/app/code/Magento/AdvancedCheckout/view/frontend/layout/customer_account.xml#L11)
- [customer-account-navigation-customer-balance-link](https://github.com/magento/partners-magento2ee/blob/2.3.2/app/code/Magento/CustomerBalance/view/frontend/layout/customer_account.xml#L10)
- [customer-account-navigation-gift-card-link](https://github.com/magento/partners-magento2ee/blob/2.3.2/app/code/Magento/GiftCardAccount/view/frontend/layout/customer_account.xml#L12)
- [customer-account-navigation-magento-invitation-link](https://github.com/magento/partners-magento2ee/blob/2.3.2/app/code/Magento/Invitation/view/frontend/layout/customer_account.xml#L12)
- [customer-account-navigation-giftregistry-link](https://github.com/magento/partners-magento2ee/blob/2.3.2/app/code/Magento/GiftRegistry/view/frontend/layout/customer_account.xml#L11)
- [customer-account-navigation-reward-link](https://github.com/magento/partners-magento2ee/blob/2.3.2/app/code/Magento/Reward/view/frontend/layout/customer_account.xml#L11)
- [customer-account-navigation-return-history-link](https://github.com/magento/partners-magento2ee/blob/2.3.2/app/code/Magento/Rma/view/frontend/layout/customer_account.xml#L11)

Thank you!

CC: @rogyar 
whatsnew
Added the "Manage the 'My Account' dashboard navigation links" section to  [Common layout customization tasks](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-manage.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 1]
```diff
@@ -475,6 +475,86 @@ class Product
 }
 ```
 
+## Manage the 'My Account' dashboard navigation links
+
+You can remove navigation links from the 'My Account' dashboard on the storefront by setting the `remove` attribute.
+
+```xml
+<!-- ################################## -->
+<!-- Magento version: Community Edition -->
```
Comment (rogyar) [inline guides/v2.3/frontend-dev-guide/layouts/xml-manage.md]: Hi @atwixfirster. That's a good clarification, however, there are Open Source and Commerce Magento versions for the time being. Please, adjust these headings. Thank you.

Comment (atwixfirster) [reply guides/v2.3/frontend-dev-guide/layouts/xml-manage.md]: **fixed**

Could you please verify now, @rogyar ?

Thank you!

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**